contracts/src/BeefyClient.sol

```diff
@@ -448,22 +448,10 @@ contract BeefyClient {
     ) internal pure returns (uint256) {
         // Start with the minimum number of signatures.
         uint256 numRequiredSignatures = minRequiredSignatures;
-
-        // We must subtract minimumSignatures from the number of validators or we might end up
-        // requiring more signatures than there are validators.
-        uint256 extraValidatorsLen = validatorSetLen.saturatingSub(minRequiredSignatures);
```

Comment:
We no longer need this `extraValidatorsLen` because we clamp the output to be less than 2/3 quorum.